src/component/Search.js

```diff
@@ -0,0 +1,64 @@
+import React,{ useState , useEffect} from 'react';
+import axios from 'axios';
+
+const Search =()=>{
+    const [term,setTerm]=useState("");
+    const [results,setResults]=useState([]);
+    const resultRender=results.map((result)=>{
+        return (
+            <div key={result.pageid} className="item">
+                 <div className="right floated content">
+                    <a className="ui button" href={`https://en.wikipedia.org?curid=${result.pageid}`}>Go</a>
+                </div>
+                <div className="content">
+                    <div className="headers">
+                        {result.title}
+                    </div>
+                    <span dangerouslySetInnerHTML={{__html:result.snippet}} ></span>
```

Comment:
What is dangerouslySetInnerHTML?

Reply:
dangerouslySetInnerHTML is used to set inner HTML for the page